docs/modules/components/pages/inputs/schema_registry.adoc

```diff
@@ -148,6 +149,15 @@ Include only subjects which match the regular expression filter. All subjects ar
 
 *Default*: `""`
 
+=== `fetch_all_schemas_on_connect`
+
+Fetch all schemas on connect. Set this to `true` when schema references are used.
+
+
+*Type*: `bool`
+
+*Default*: `true`
+
```

Comment:
This name is not very intuitive for me.

Can we name it:
```
enforce_ordering: bool

If true all schemas will be fetched and sorted, so that schemas are inserted in the same order. This is required to correctly resolve schema references.

default: true
```

On another note: do we even need this option? Is this only false if we would OOM reading all the schemas?

Reply:
Or maybe `fetch_in_order` since they can still mess up the order on the output with max_in_flight

Reply:
Naming things is hard 😅 I like `fetch_in_order`, thanks!

Reply:
> Is this only false if we would OOM reading all the schemas?

I'm not super-concerned about OOM unless the schemas are huge, but it could take a long time to fetch them all if there are many of them, so it might be handy to stream them if people wish to stick a cache in the middle. Or maybe I'm overthinking it... The tests cover both flows now, so I'm happy to leave it there.